### PR TITLE
docs: update readme logo

### PR DESCRIPTION
**Pull request by aayushmau5**
**Description**

- Update the `README.md` logo/banner

**Related issue(s)**
Resolves https://github.com/asyncapi/diff/issues/89

### CODE REVIEW

**Review by derberg — APPROVED** [review]

**Comment by derberg** [comment]
closed and reopened to see if we can trigger handing ci

File: README.md [hunk 1]
```diff
@@ -1,13 +1,6 @@
...
-  <a href="https://www.asyncapi.org"><img src="https://github.com/asyncapi/parser-nodejs/raw/master/assets/logo.png" alt="AsyncAPI logo" width="200"></a>
-  <br>
-  Diff
-</h5>
-<p align="center">
-  <em>
-  AsyncDiff is a library that compares two AsyncAPI files and provides information about the differences by pointing out explicitly information like breaking changes.
-  </em>
-</p>
+[![AsyncAPI Banner and Logo](/assets/logo.png)](https://www.asyncapi.com)
```
Comment (magicmatatjahu) [inline README.md:1]:
```suggestion
[![AsyncAPI Diff](/assets/logo.png)](https://www.asyncapi.com)
```

**Review by magicmatatjahu — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (mcturco) [reply README.md:1]: + 1 on this suggestion

**Review by magicmatatjahu — APPROVED** [review]

**Review by onbit-uchenik — APPROVED** [review]

**Pull request merged**